doc/release-process.md

```diff
@@ -0,0 +1,254 @@
+Release Process
+====================
+
+Before every release candidate:
+
+* Update version in `configure.ac` (don't forget to set `CLIENT_VERSION_IS_RELEASE` to `true`)
+* Write release notes to changelog
+
+Before every major release:
+
+* Update hardcoded, see [this pull request](https://github.com/gridcoin/Gridcoin-Research/pull/783) for an example.
+* Update version of `contrib/gitian-descriptors/*.yml`: usually one'd want to do this on master after branching off the release - but be sure to at least do it before a new major release
+
+### First time / New builders
+
+If you're using the automated script (found in [contrib/gitian-build.sh](/contrib/gitian-build.sh)), then at this point you should run it with the "--setup" command. Otherwise ignore this.
+
+Check out the source code in the following directory hierarchy.
+
+    cd /path/to/your/toplevel/build
+    git clone https://github.com/gridcoin-community/gitian.sigs.git
+    git clone https://github.com/bitcoin-core/bitcoin-detached-sigs.git
+    git clone https://github.com/devrandom/gitian-builder.git
+    git clone https://github.com/gridcoin/Gridcoin-Research.git
+
+### Bitcoin maintainers/release engineers, suggestion for writing release notes
+
+Write release notes. git shortlog helps a lot, for example:
+
+    git shortlog --no-merges v(current version, e.g. 0.7.2)..v(new version, e.g. 0.8.0)
+
+Generate list of authors:
+
+    git log --format='%aN' "$*" | sort -ui | sed -e 's/^/- /'
+
+Tag version (or release candidate) in git
+
+    git tag -s v(new version, e.g. 0.8.0)
```

Comment:
We don't use the "v" prefix in the 3.x tags.

Reply:
Yes, this should already be reflected in the actual gitian script. I'll change it, when I'm able to compile an actual gitian.